re/bindings.go

```diff
@@ -15,6 +16,67 @@ import (
 	lua "github.com/yuin/gopher-lua"
 )
 
+func (re *re) luaDecrypt(l *lua.LState) int{
+	key_str := l.ToString(1)
+	iv_str := l.ToString(2)
+	enc_str := l.ToString(3)
+
+	key, err := hex.DecodeString(key_str)
+	if err != nil {
+		l.RaiseError("Failed to decode Key: %v", err)
+		return 0
+	}
+
+	iv, err := hex.DecodeString(iv_str)
+	if err != nil {
+		l.RaiseError("Failed to decode IV: %v", err)
+		return 0
+	}
+
+	enc, err := hex.DecodeString(enc_str)
+	if err != nil {
+		l.RaiseError("Failed to decode Encrypt: %v", err)
+		return 0
+	}
+
+	dec := decrypt(key, iv, enc)
+	if err != nil {
+		l.RaiseError("Failed to decrypt: %v", err)
+		return 0
+	}
+	
+	l.Push(lua.LString(hex.EncodeToString(dec)))
+	return 1	
+}
+
+func (re *re) luaEncrypt(l *lua.LState) int {
```

Comment:
No need to make this RE function, this can be a standalone (`func LuaEncrypt(l *lua.LState) int`).